charts/nextcloud/templates/deployment.yaml

```diff
@@ -305,7 +304,27 @@ spec:
           - "-c"
           - "until pg_isready -h ${POSTGRES_HOST} -U ${POSTGRES_USER} ; do sleep 2 ; done"
       {{- end }}
-      {{- end }}
+      - name: {{ .Chart.Name }}-upgrade
+        image: {{ include "nextcloud.image" . }}
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        args:
+          # Do upgrade and then exit
+          # This avoids doing it in the container, which could get killed by failed liveness probes
+          - "true"
+        env:
+        - name: NEXTCLOUD_UPDATE
+          value: "1"
+{{- include "nextcloud.env" . | indent 8 }}
+        resources:
+{{ toYaml .Values.resources | indent 10 }}
```

Comment:
```suggestion
        {{- include "nextcloud.env" . | indent 8 }}
        resources:
        {{ toYaml .Values.resources | indent 10 }}
```

Reply:
Same as previous. This is the syntax used in the current container.